server/constants.py

```diff
@@ -24,12 +24,15 @@ class PluginSettings:
     HUB_PUB_KEY = 'wholetale.pub_key'
     INSTANCE_CAP = 'wholetale.instance_cap'
     DATAONE_URL = 'wholetale.dataone_url'
+    DATAVERSE_URL = 'wholetale.dataverse_url'
 
 
 class SettingDefault:
     defaults = {
         PluginSettings.INSTANCE_CAP: 2,
-        PluginSettings.DATAONE_URL: 'https://cn.dataone.org/cn/v2/node'
+        PluginSettings.DATAONE_URL: 'https://cn.dataone.org/cn/v2/node',
+        PluginSettings.DATAVERSE_URL:
+            'https://services.dataverse.harvard.edu/miniverse/map/installations-json'
```

Comment:
This "miniverse" URL may change some day but we'll try to let you know if it does. If we forget and it changes under you, please email support@dataverse.org about it.